src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -391,7 +391,9 @@ $(function () {
               }
             }
             if (isActive) {
-              $(e).addClass(active);
+              if ((window.location.pathname != '/index.html') && (window.location.pathname != '/')) {
+                $(e).addClass(active);
```

Comment:
The e2e test in CI fails because the breadcrumb is missing for index.html, which should be related to this piece of code.

I think the proper fix should modify L389 `if (util.getAbsolutePath(href) === currentAbsPath)`. This line checks whether the node in nav bar links to the current page. In the drop down menu scenario, it happens `getAbsolutePath("#") === "index.html"`, which makes `isActive = true` wrongly. 

What about complement this check to make sure the current node isn't a drop down placeholder? This may be done by check whether the node has the attribute `data-toggle="dropdown"`.

Reply:
This has been moved in 2c68a28